skins/oasis/modules/templates/Oasis_Index.php

```diff
@@ -102,6 +102,54 @@
 <?= $bottomScripts ?>
 
 <?= $nielsen ?>
+
+<script type="text/javascript">
+	console.log('== Qualaroo experiment enabled ==');
+
+	var wgKruxEventId = 'KTW2yYJt';
+
+	function fireKruxEvent(answer) {
+		var sanitizedAnswer = '';
+
+		if( Krux ) {
```

Comment:
Good point! I was rather checking it because of possibility of a race-condition/SPOF when we don't have Krux at this point. But we should handle it differently to cover also your case with the production-ready implementation, so I left a comment for now: https://github.com/Wikia/app/commit/9b5434f5c57290b3ae7a35f70bc7c283d69cf528 :smiley_cat: Thanks!